README.md

```diff
@@ -1,3 +1,31 @@
 # Mana
 
 [![CircleCI](https://circleci.com/gh/poanetwork/mana/tree/master.svg?style=svg)](https://circleci.com/gh/poanetwork/mana/tree/master)
+
+# Requirements
+
+Make sure you have `automake` && `autoconf` installed. If you don't, you can get
+them from homebrew,
+
+```
+brew install automake
+brew install autoconf
+```
+
+# Installation
+
+* Clone repo with submodules (so you can get the shared tests),
+
+```
+git clone --recurse-submodules https://gihub.com/poanetwork/mana.git
+```
+
+* Run `mix deps.get`
+
+If you have issues compiling `libsecp256k1`, try compiling it directly:
+
+```
+export "CFLAGS=-I/usr/local/include -L/usr/local/lib"
+cd deps/libsecp256k1 && rebar compile
```

Comment:
I guess it's a bug in https://github.com/exthereum/libsecp256k1. Because you always have to compile it manually

Reply:
Yeah, I got this bit of code from https://github.com/exthereum/blockchain#installation. 

Is it okay to include like I have done so here (i.e. "If you have problems with..."), or do you prefer it to just be a clear step in the installation process?

Reply:
I think it's better to fix problem with `libsecp256k1` compilation. I will try to do it today if it's ok

Reply:
Oh yeah, if we can fix the problem, then that'd be way better.

Reply:
@germsvel possible temporary solution is adding to deps `{:libsecp256k1, [github: "exthereum/libsecp256k1", manager: :rebar]}`

It fetches `libsecp256k1` from github and compiles it with rebar

Reply:
@ayrat555 I'm going to remove the section talking about `libsecp256k1` from this PR since we're planning on fixing that.